_docs/3.0.0/usage/menas-api.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: docs
+title: Usage - Menas API
+version: '3.0.0'
+categories:
+    - '3.0.0'
+    - usage
+---
+
+- [Export Entity](#export-entity)
+  - [Example Import Response for Schema](#example-import-response-for-schema)
+- [Entity Import](#entity-import)
+  - [JSON payload](#json-payload)
+  - [Example Export Response for Schema](#example-export-response-for-schema)
+
+
+### Export Entity
```

Comment:
I would personally not include this page at all, since we have hidden all V2 API in non-dev deployments. I assume that the V2.0.0 documentation still contains this, but we have agreed to include V2 API only for compatibility reasons for a limited period of time.

Of course, the V3 API should be present in the documentation in the end (after this PR is merged), but it should not be called _Menas API_ anymore, I presume.